BootloaderCommonPkg/Library/IasImageLib/IasImage.c

```diff
@@ -13,6 +13,7 @@
 #include <Library/DebugLib.h>
 #include <Library/Crc32Lib.h>
 #include <Library/SecureBootLib.h>
+#include <Library/MemoryAllocationLib.h>

```

Comment:
Why do we add MemoryAllocationLib.h ?

Reply:
Added as part of previous implementation. Will remove this header.